### PR TITLE
feat(i18n): add slovenian language

### PR DESCRIPTION
**Pull request by ariskemper**

### CODE REVIEW

**Comment by fabian-hiller** [comment]
Thank you for this PR! Do you know another native speaking developer that could review the translations for me?

File: packages/i18n/src/sl.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    length:         (issue) => `Neveljaven dolžina: Pričakovana ${issue.expected}, vendar prejeta ${issue.received}`,
+    mac:            (issue) => `Neveljaven MAC: Prejet ${issue.received}`,
+    mac48:          (issue) => `Neveljaven 48-bit MAC: Prejet ${issue.received}`,
+    mac64:          (issue) => `Neveljaven 64-bit MAC: Prejet ${issue.received}`,
+    maxBytes:       (issue) => `Neveljaven bajti: Pričakovani ${issue.expected}, vendar prejeti ${issue.received}`,
+    maxLength:      (issue) => `Neveljaven dolžina: Pričakovana ${issue.expected}, vendar prejeta ${issue.received}`,
+    maxSize:        (issue) => `Neveljaven velikost: Pričakovana ${issue.expected}, vendar prejeta ${issue.received}`,
+    maxValue:       (issue) => `Neveljaven vrednost: Pričakovana ${issue.expected}, vendar prejeta ${issue.received}`,
+    mimeType:       (issue) => `Neveljaven MIME type: Pričakovan ${issue.expected}, vendar prejet ${issue.received}`,
+    minBytes:       (issue) => `Neveljaven bytes: Pričakovan ${issue.expected}, vendar prejet ${issue.received}`,
```
Comment (mkosir) [inline packages/i18n/src/sl.ts:45]: Maybe we could wrote
```suggestion
    minBytes:       (issue) => `Neveljaven bajt: Pričakovan ${issue.expected}, vendar prejet ${issue.received}`,
```

**Review by mkosir — APPROVED** [review]
Looks nice 👍 
Used our [IT dictionary](https://dis-slovarcek.ijs.si/) to check naming, since have to admit some "tech" words does sound funny in Slovenian :)

**Comment by fabian-hiller** [comment]
Thank you!

**Pull request merged**